src/openclsim/model/basic_activity.py

```diff
@@ -11,12 +11,20 @@ class BasicActivity(GenericActivity):
     BasicActivity Class is a generic class to describe an activity, which does not
     require any specific resource, but has a specific duration.
 
-    duration: time required to perform the described activity.
-    additional_logs: list of other concepts, where the start and the stop of the basic
-    activity should be recorded.
-
-    start_event: the activity will start as soon as this event is processed
-                 by default will be to start immediately
+    Parameters
+    ----------
+    duration
+        time required to perform the described activity.
+    name
```

Comment:
Is this used?

Reply:
I just did cosmetics

Reply:
Yes durations of basic activities are used quite a lot.